alacritty_terminal/src/term/mod.rs

```diff
@@ -1959,24 +1818,22 @@ impl<T: EventListener> ansi::Handler for Term<T> {
             Attr::Reset => {
                 self.cursor.template.fg = Color::Named(NamedColor::Foreground);
                 self.cursor.template.bg = Color::Named(NamedColor::Background);
-                self.cursor.template.flags = cell::Flags::empty();
-            },
-            Attr::Reverse => self.cursor.template.flags.insert(cell::Flags::INVERSE),
-            Attr::CancelReverse => self.cursor.template.flags.remove(cell::Flags::INVERSE),
-            Attr::Bold => self.cursor.template.flags.insert(cell::Flags::BOLD),
-            Attr::CancelBold => self.cursor.template.flags.remove(cell::Flags::BOLD),
-            Attr::Dim => self.cursor.template.flags.insert(cell::Flags::DIM),
-            Attr::CancelBoldDim => {
-                self.cursor.template.flags.remove(cell::Flags::BOLD | cell::Flags::DIM)
+                self.cursor.template.flags = Flags::empty();
             },
-            Attr::Italic => self.cursor.template.flags.insert(cell::Flags::ITALIC),
-            Attr::CancelItalic => self.cursor.template.flags.remove(cell::Flags::ITALIC),
-            Attr::Underscore => self.cursor.template.flags.insert(cell::Flags::UNDERLINE),
-            Attr::CancelUnderline => self.cursor.template.flags.remove(cell::Flags::UNDERLINE),
-            Attr::Hidden => self.cursor.template.flags.insert(cell::Flags::HIDDEN),
-            Attr::CancelHidden => self.cursor.template.flags.remove(cell::Flags::HIDDEN),
-            Attr::Strike => self.cursor.template.flags.insert(cell::Flags::STRIKEOUT),
-            Attr::CancelStrike => self.cursor.template.flags.remove(cell::Flags::STRIKEOUT),
+            Attr::Reverse => self.cursor.template.flags.insert(Flags::INVERSE),
+            Attr::CancelReverse => self.cursor.template.flags.remove(Flags::INVERSE),
+            Attr::Bold => self.cursor.template.flags.insert(Flags::BOLD),
+            Attr::CancelBold => self.cursor.template.flags.remove(Flags::BOLD),
+            Attr::Dim => self.cursor.template.flags.insert(Flags::DIM),
+            Attr::CancelBoldDim => self.cursor.template.flags.remove(Flags::BOLD | Flags::DIM),
+            Attr::Italic => self.cursor.template.flags.insert(Flags::ITALIC),
+            Attr::CancelItalic => self.cursor.template.flags.remove(Flags::ITALIC),
+            Attr::Underscore => self.cursor.template.flags.insert(Flags::UNDERLINE),
+            Attr::CancelUnderline => self.cursor.template.flags.remove(Flags::UNDERLINE),
```

Comment:
Would you mind to change it from `Underscore` to `Underline`, since we have a little inconsistency right now, and you've already touched this lines?

Reply:
I'm not sure why we've chosen `Underscore` here, I'll take a look at changing this, thanks.

It's possible that this naming comes from a spec or other docs though, but I don't think I've written this myself.